main.py

```
@@ -0,0 +1,54 @@
+
+bot = Bot(token=TELEGRAM_TOKEN, parse_mode=ParseMode.HTML)
+dp = Dispatcher()
+app = FastAPI()
+
+@app.get("/health")
+async def health():
+    return {"status": "ok"}
+
+async def save_file(file_id: str, suffix: str) -> str:
+    f = await bot.get_file(file_id)
+    fd, path = tempfile.mkstemp(suffix=suffix); os.close(fd)
```

Ruff (0.12.2)

Multiple statements on one line (semicolon)

(E702)